schema/workflow/unit/condition.json

```diff
@@ -46,6 +46,7 @@
         "input",
         "statement",
         "then",
-        "else"
+        "else",
+        "maxOccurrences"
```

Comment:
We could also put a default value (e.g. 100) for this and not require it.

Reply:
adjusted